VERSION

```diff
@@ -1 +1 @@
-1.13.0
+1.13.1-dev
```

Comment:
What is this?

Reply:
I pulled the changes from envoyproxy#10025 manually